src/planning/plan/tests/plan-consumer-test.ts

```diff
@@ -10,22 +10,59 @@
 import '../../../runtime/storage/firebase/firebase-provider.js';
 import '../../../runtime/storage/pouchdb/pouch-db-provider.js';
 import {assert} from '../../../platform/chai-web.js';
+import {Loader} from '../../../platform/loader.js';
+import {Manifest} from '../../../runtime/manifest.js';
+import {Arc} from '../../../runtime/arc.js';
+import {Id} from '../../../runtime/id.js';
 import {Modality} from '../../../runtime/modality.js';
 import {Relevance} from '../../../runtime/relevance.js';
 import {MockSlotComposer} from '../../../runtime/testing/mock-slot-composer.js';
-import {PlanningTestHelper} from '../../testing/planning-test-helper.js';
 import {PlanConsumer} from '../../plan/plan-consumer.js';
 import {Planificator} from '../../plan/planificator.js';
 import {PlanningResult} from '../../plan/planning-result.js';
 import {Suggestion} from '../../plan/suggestion.js';
-import {PlanningModalityHandler} from '../../planning-modality-handler.js';
 import {SuggestFilter} from '../../plan/suggest-filter.js';
-
-async function createPlanConsumer(arcKey, storageKeyBase, helper) {
-  helper.arc.storageKey = 'volatile://!158405822139616:demo^^volatile-0';
-  const store = await Planificator['_initSuggestStore'](helper.arc, storageKeyBase);
+import {Planner} from '../../planner.js';
+import {RecipeIndex} from '../../recipe-index.js';
+import {Speculator} from '../../speculator.js';
+import {PlanningModalityHandler} from '../../planning-modality-handler.js';
+import {devtoolsArcInspectorFactory} from '../../../devtools-connector/devtools-arc-inspector.js';
+import {devtoolsPlannerInspectorFactory} from '../../../devtools-connector/devtools-planner-inspector.js';
+
+async function setup(slotComposer, storageKeyBase, manifestString) {
+  Planner.clearCache();
+  const loader = new Loader();
+  const context = await Manifest.parse(manifestString, {loader, fileName: ''});
+  const arc = new Arc({
+    id: Id.fromString('demo'),
+    slotComposer,
+    loader,
+    context,
+    storageKey: 'volatile://!158405822139616:demo^^volatile-0',
+    inspectorFactory: devtoolsArcInspectorFactory
+  });
+  const recipeIndex = RecipeIndex.create(arc);
```

Comment:
recipeIndex isn't needed for this test either. there is a helper method: `StrategyTestHelper.createTestStrategyArgs` you could use to set the planner init parameters